.github/workflows/windows.yml

```diff
@@ -15,8 +15,13 @@ jobs:
         uses: actions/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
+      - name: DevKit, ENV['PATH']
+        uses: MSP-Greg/msys2-action@master
```

Comment:
What is this?

Reply:
@bronzdoc

Given that the requirements here are minor, removed the custom action and replaced with `ridk enable` in the test script.